state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -191,6 +192,10 @@ pub mod pallet {
 			AttemptCount = AttemptCount,
 		>;
 
+		type ElectionEgressWitnesser: WatchForEgressSuccess<
```

Comment:
nit: might as well use the same names? ie. name the trait `ElectionEgressWitnesser`.